lib/versions.js

```diff
@@ -0,0 +1,81 @@
+const chalk = require("chalk");
+
+/**
+ * Prints the versions overview
+ */
+class Versions
```

Comment:
I understand the purpose of this class though the name and especially the passed constructor arguments make me assume it does something completely different than what the name intents.

Can we rename this class to something like `(Webpack)PackageVersionPrinter`?

Reply:
This code was completely refactored in the meantime.